x/authz/types/keys.go

```diff
@@ -0,0 +1,41 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "authz"
+
+	// StoreKey is the store key string for authz
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for authz
+	RouterKey = ModuleName
+
+	// QuerierRoute is the querier route for authz
+	QuerierRoute = ModuleName
+)
+
+// Keys for authz store
+// Items are stored with the following key: values
+//
+// - 0x01<granterAddress_Bytes><granteeAddress_Bytes><msgType_Bytes>: Grant
+
+var (
+	// Keys for store prefixes
+	GrantKey = []byte{0x01} // prefix for each key
+)
+
+// GetActorAuthorizationKey - return authorization store key
+func GetActorAuthorizationKey(grantee sdk.AccAddress, granter sdk.AccAddress, msgType string) []byte {
```

Comment:
I'm still not 100% convinced about the "actor" naming, unless I'm missing the context

Reply:
My bad, will update this. Looks like there are a couple of places where it is still used as `actor`